packages/cli/src/config/config.ts

```diff
@@ -737,7 +737,8 @@ export async function loadCliConfig(
     interactive,
     trustedFolder,
     useRipgrep: settings.tools?.useRipgrep,
-    shouldUseNodePtyShell: settings.tools?.shell?.enableInteractiveShell,
+    shouldUseNodePtyShell:
```

Comment:
Should update this to also be user facing. That way users trying to utilize AI to make changes will be more successful